### ISSUE TITLE
update the latest version on cdnjs

### ISSUE THREAD

**Issue by kennynaoh** [DataTables/FixedHeader#83]
Hello @DataTables ,
I am the member of cdnjs project.
We want to update the latest version on [cdnjs](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/datatables-fixedheader).
I want to make sure does this project publish on npm?
I saw many similar npm name about this project, like [this](https://www.npmjs.com/package/datatables.net-fixedheader).
Please help me confirm that what npm should we use if published or I just update it to the latest version by [github](https://github.com/DataTables/FixedHeader).
Thanks for your help!

https://github.com/cdnjs/cdnjs/issues/8742

**Comment by DataTables** [comment]
Hi,

Thanks for your question. This is a source repo and isn't really suitable for inclusion on cdnjs. Instead, I would recommend you include my distribution repos:

* [FixedHeader core](https://github.com/DataTables/Dist-DataTables-FixedHeader)
* [FixedHeader DataTables styling](https://github.com/DataTables/Dist-DataTables-FixedHeader-DataTables) - npm: datatables.net-fixedheader
* [FixedHeader DataTables styling](https://github.com/DataTables/Dist-DataTables-FixedHeader-DataTables) - npm: datatables.net-fixedheader-dt
* [FixedHeader Bootstrap styling](https://github.com/DataTables/Dist-DataTables-FixedHeader-Bootstrap) - npm: datatables.net-fixedheader-bs
* [FixedHeader Foundation styling](https://github.com/DataTables/Dist-DataTables-FixedHeader-Foundation) - npm: datatables.net-fixedheader-zf
* [FixedHeader jQuery UI styling](https://github.com/DataTables/Dist-DataTables-FixedHeader-jQueryUI) - npm: datatables.net-fixedheader-jqui

You could include this repo, but will usually require a compilation step to include a source repo (for example I don't have CSS files in this repo, only SCSS).

The reason there isn't a `dist` directory in this repo is for npm. Because DataTables and its extensions support multiple styling libraries, to include the required styles without needing to manually enter paths I had to split it into multiple distribution repos (which is a bit of a pain...).

**Comment by kennynaoh** [comment]
Sorry for late reply.
So if I want to update the new version of this library(https://cdnjs.cloudflare.com/ajax/libs/datatables-fixedheader/2.1.1/dataTables.fixedHeader.js) on npm.
Could you give me some advices?
Thanks!

**Comment by DataTables** [comment]
I'd say that particular version shouldn't really be there at all - `datatables.net-fixedheader` is the deploy `npm` package as I mentioned above. Can you alias that package to your `datatables-fixedheader` one?

**Comment by kennynaoh** [comment]
Are they the same library?

**Comment by DataTables** [comment]
Yes. As I tried to explain above (perhaps not very well!), the distribution repos are just built versions of this repo, split apart so you can choose which styling framework you want to use with it.

This repo contains the source files that are compiled and separated into the various distribution repos (a core Javascript repo and one for each of the styling frameworks that DataTables supports).

**Comment by kennynaoh** [comment]
Because there is exist [datatables-fixedheader](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/datatables-fixedheader) lib on [cdnjs](https://cdnjs.com/libraries/datatables-fixedheader).
And I want to update it to the latest version on npm(if has) and add npm auto-update config.
So you mean if there is new version publish on git repo, and it will also publish on npm [here](https://www.npmjs.com/package/datatables.net-fixedheader)?
I just want to make sure about that.
Thanks for your help!

**Comment by DataTables** [comment]
Yes, `datatables-fixedheader` is this repo. However, while making all of the extensions available with the various styling frameworks I had to split the compiled files into various repos. Rather than storing them both in this repo and the distribution repos, I keep them only in the distribution repo.

Personally I would say that `datatables-fixedheader` should track this repo - even although it is redundant and doesn't contain the compiled files.

Your users should use `datatables.net-fixedheader` (and the various styling packages for it) to track the deploy files.
